setuptools/independent/RuntimeGraphGenerator/RuntimeGraphManager.cs

```diff
@@ -24,8 +24,8 @@ public NuGet.RuntimeModel.RuntimeGraph Collect(IEnumerable<LibraryExport> export
                     if (runtimeJson != null)
                     {
                         // Convert the package-name to lower-case in the path to lookup runtime.json
-                        string lowercasedPackageName = description.Identity.Name.ToLower();
-                        string pathToPackage = export.Library.Path.Replace(description.Identity.Name, lowercasedPackageName);
+                        // string lowercasedPackageName = description.Identity.Name.ToLower();
```

Comment:
the packages we produces have both upper and lower case characters, this worked on windows and mac but not in linux. Eventaully all tooling and packaging will move to lowercase but for now we just avoid lower casing ourselves.